packages/holocron/__tests__/loadModule.web.spec.js

```diff
@@ -16,15 +16,21 @@ import { fromJS } from 'immutable';
 import loadModule from '../src/loadModule.web';
 import { resetModuleRegistry } from '../src/moduleRegistry';
 
+const map = {};
```

Comment:
might be better to name the variable by what it's a map of? looks like a bag of registered mock event listeners, maybe `mockElementListeners`?

Reply:
looking again, it appears listeners are called via `mockElement.addEventListener.mock.calls[0][1](..)`, is this `map` var needed? (maybe left over from debugging?)

Reply:
using it on line `31` to populate the list of events listeners to be added to the mock. Let me know if  that is what you meant?

Reply:
cb functions are stored in `map`, but are they accessed/used?

Reply:
yes i 'm using this to mock the cb functions within the test, not sure that answers the questions